src/Components/MainTableView/DeleteTuple/DeleteTuple.tsx

```diff
@@ -3,32 +3,32 @@ import "./DeleteTuple.css";
 import TableAttribute from '../DataStorageClasses/TableAttribute';
 import TableAttributesInfo from '../DataStorageClasses/TableAttributesInfo';
 import TableAttributeType from '../enums/TableAttributeType';
-
-import CheckDependency from '../CheckDependency';
 import PrimaryTableAttribute from '../DataStorageClasses/PrimaryTableAttribute';
 import SecondaryTableAttribute from '../DataStorageClasses/SecondaryTableAttribute';
 
-/**
- * list of allowed states on this delete tuple component
- */
-type deleteTupleState = {
-  dependencies: Array<any>, // list of dependencies fetched from API
-  deleteStatusMessage: string, // for GUI to show
-  isGettingDependencies: boolean, // for loading animation status
-  deleteAccessible: boolean // valdiation result of accessibility from check dependency component
+
+interface DeleteTupleProps {
+  token: string;
+  selectedSchemaName: string;
+  selectedTableName: string;
+  tableAttributesInfo?: TableAttributesInfo;
+  selectedTableEntry?: any;
```

Comment:
`any` use

Reply:
That has to be any cause the object passing back has different attirbutes depending on what table is being selected, thus there is no nice way to restrict the type.